<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareWidget</class>
 <widget class="QWidget" name="HardwareWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QPushButton" name="serialTogglebutton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Port :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="portBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="grblSettingsButton">
     <property name="text">
      <string>Grbl Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Baudrate :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="baudrateBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Status :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="grblStateLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
